package.json

```diff
@@ -120,6 +120,9 @@
     "@ngrx/store": "^17.1.1",
     "@ngx-translate/core": "^14.0.0",
     "@nicky-lenaers/ngx-scroll-to": "^14.0.0",
+    "@terraformer/wkt": "^2.2.1",
+    "@types/grecaptcha": "^3.0.4",
+    "angular-idle-preload": "3.0.0",
```

Comment:
I think we removed `@types/grecaptcha` and `angular-idle-preload` a while back from `main` as they were unused.  I don't think they are used here either (unless I missed it), so they can be removed from this PR

Reply:
done